### PR TITLE
Correct zendesk URL, include userAgent in details

### PR DESCRIPTION
**Pull request by nathanstitt**
Tweaks the wording a bit to remove the mention of "Server" since it also reports JS errors.

screen shot 2016-01-13 at 10 17 16 am

### CODE REVIEW

File: src/concept-coach/error-notification.cjsx [hunk 1]
```diff
@@ -43,20 +43,25 @@ ErrorNotification = React.createClass
...
-            href="https://openstaxtutor.zendesk.com/hc/en-us/requests/new"
+            href="https://openstaxcc.zendesk.com/hc/en-us"
           > our support site </a> so we can help to diagnose and correct the issue.
         </p>
-        <BS.Button onClick={@toggleDetails}>
+        <p>
+          When reporting the issue, it would be helpful if you could include the error details.
+        </p>
+        <BS.Button className='-display-errors' style={marginBottom: '1rem'} onClick={@toggleDetails}>
           {if @state.isShowingDetails then "Hide" else "Show"} Details
```
Comment (nathanstitt) [inline src/concept-coach/error-notification.cjsx:65]: @pandafulmanda & @philschatz : What's the opinion of using inline styles like this?  I felt like it was too heavy to create a `concept-coach/error-notification.less` stylesheet, but also didn't want to throw the margin rule in with an a bunch of unrelated ones.

Comment (philschatz) [inline src/concept-coach/error-notification.cjsx:65]: maybe since it's just one style, add it to the main CC stylesheet?

**Comment by philschatz** [comment]
Nice! Anything else we should try to include in the screenshot?

Also, can this be a hotfix?

File: src/concept-coach/error-notification.cjsx (hunk 1 above)
Comment (nathanstitt) [inline src/concept-coach/error-notification.cjsx:65]: Yeah, we could do that but then it's a stray style hanging out there and will be forgotten.  Then when/if we ever remove the button the style won't be remembered and will remain in the main stylesheet forever.  

There's tons of opinions on the validity of using inlines styles with react.  I'm slowly coming to the opinion that inline styles are a good for a single style like the above when the component doesn't have an associated stylesheet.  When the component gets more than one or two inline styles then they should be copied to a stylesheet.

But like I said, I'm still in flux on the subject, and I pointed it out was to gather other viewpoints.

Comment (philschatz) [inline src/concept-coach/error-notification.cjsx:65]: Let's try it out; I just wish we had a `CONVENTIONS.md` file somewhere that we could update with this idea...

**Pull request merged**